crates/holochain_sqlite/src/db.rs

```diff
@@ -160,6 +171,12 @@ impl DbWrite {
     }
 }
 
+pub fn num_read_threads() -> usize {
+    let num_cpus = num_cpus::get();
+    let num_threads = num_cpus.checked_div(2).unwrap_or(0);
```

Comment:
Do we really want `0` or should the minimum be `1`?

Reply:
See the next line down, it takes the min of this and 4

Reply:
ah, gottcha